pynecone/app.py

```diff
@@ -316,6 +321,34 @@ def get_load_events(self, route: str) -> List[EventHandler]:
             route = constants.INDEX_ROUTE
         return self.load_events.get(route, [])
 
+    def add_download_file(self, download_url: str, file_path: str):
+        """Download file from host by downloa_durl.
+
+        Args:
+            download_url: The route to get the file.
+            file_path: The file from host.
+
+        Returns:
+            Respone a file from server.
+
+        Raises:
+            ValueError: download_url already is used.
+        """
+        if download_url in self.backend_router:
```

Comment:
probably also need to check in `self.pages` for practical purposes.

Reply:
pages seems to be used for frontend so I don't check this part.
If Im wrong, please tell me.

Reply:
i think a common pattern for deployments is to use a reverse proxy that forwards requests to either frontend or backend based on the path, so there's a possibility of conflict in these cases. maybe it's not a big deal though.

Reply:
I used self.add _page to redirect frontend to backend and there will check self.page.